<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_gen_gold">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>generate for gold</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_gen_iron_wood">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>generate for iron and wood</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
